### PR TITLE
[JBTM-3605] Update jaxws-ri version from 2.3.0 to 2.3.5

### PR DESCRIPTION
**Pull request by marcosgopen**
Update jaxws-ri dependency version from 2.3.0 to 2.3.5 https://issues.redhat.com/browse/JBTM-3605

JDK11 CORE AS_TESTS XTS !TOMCAT !RTS !JACOCO !QA_JTA !QA_JTS_JACORB !QA_JTS_JDKORB !QA_JTS_OPENJDKORB !PERF !LRA !DB_TESTS !mysql !db2 !postgres !oracle

### CODE REVIEW

File: XTS/pom.xml [hunk 1]
```diff
@@ -80,7 +80,7 @@
              <jvm.args.modular>--add-modules=java.se</jvm.args.modular>
 	     <version.org.jboss.spec.javax.xml.bind.jboss-jaxb-api_2.3_spec>1.0.1.Final</version.org.jboss.spec.javax.xml.bind.jboss-jaxb-api_2.3_spec>
 	     <version.org.jboss.spec.javax.xml.ws.jboss-jaxws-api_2.3_spec>1.0.0.Final</version.org.jboss.spec.javax.xml.ws.jboss-jaxws-api_2.3_spec>
-	     <version.jaxws-ri>2.3.0</version.jaxws-ri>
+	     <version.jaxws-ri>2.3.5</version.jaxws-ri>
```
[the remark this answers is not in the document]
Comment (marcosgopen) [reply XTS/pom.xml:83]: Thank you Manuel for your review. I think is a good idea!

**Comment by mmusgrov** [comment]
> FYI, the core tests fail due to JBTM-3604 issue. I would suggest to close PR about JBTM-3604 and then retest this PR.

@marcosgopen Agreed, I have just approved it so when you've merged #1977 we should trigger a retest of this PR.

**Comment by mmusgrov** [comment]
@marcosgopen So I would choose:

```
JDK11 CORE AS_TESTS XTS !TOMCAT !RTS !JACOCO !QA_JTA !QA_JTS_JACORB !QA_JTS_JDKORB !QA_JTS_OPENJDKORB !PERF !LRA !DB_TESTS !mysql !db2 !postgres !oracle
```

**Comment by marcosgopen** [comment]
Thank you @mmusgrov for the review! I updated the comment.

**Comment by mmusgrov** [comment]
> RTS profile tests failed (http://narayanaci1.eng.hst.ams2.redhat.com/job/btny-pulls-narayana/PROFILE=RTS,jdk=jdk8.latest,label=linux/773/): Requested JDK version 8 cannot run with axis RTS: please use jdk 11 instead

@marcosgopen the combination of the CI script (narayana.sh) and the Jenkins job config for the pull job is inconsistent so you need to tell it what to run and what not to run so to avoid JDK8 builds you need to explicitly include `!JDK8` in the pull description.

**Comment by marcosgopen** [comment]
TESTIT

**Comment by marcosgopen** [comment]
> AS_TESTS profile tests failed (http://narayanaci1.eng.hst.ams2.redhat.com/job/btny-pulls-narayana/PROFILE=AS_TESTS,jdk=jdk11.latest,label=linux/778/): AS tests failed

@mmusgrov I have just noticed there are AS_TESTS errors. Are they PR related or I can merge the PR?

**Comment by jmfinelli** [comment]
LGTM :+1: Thanks Marco

**Pull request merged**